core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -52,12 +46,11 @@
 class CloudSqlInstance {
 
   private static final Logger logger = Logger.getLogger(CloudSqlInstance.class.getName());
-  private static final String SQL_LOGIN_SCOPE = "https://www.googleapis.com/auth/sqlservice.login";
 
   private final ListeningScheduledExecutorService executor;
-  private final SqlAdminApiFetcher apiFetcher;
+  private final InstanceDataSupplier apiFetcher;
```

Comment:
Let's always match field names to type names, either exactly or as a shorter version, e.g., instanceSupplier.

Reply:
Fixed.